docs/user-interface/controls/blazorwebview.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "BlazorWebView"
+description: "The .NET Multi-platform App UI (.NET MAUI) BlazorWebView control enables you to host a Blazor web app in your .NET MAUI app, and integrate the app with device features."
+ms.date: 08/19/2021
+---
+
+# BlazorWebView
+
+The .NET Multi-platform App UI (.NET MAUI) `BlazorWebView` is a control that enables you to host a Blazor web app in your .NET MAUI app. These apps, known as .NET MAUI Blazor apps, enable a Blazor web app to be integrated with native platform features and UI controls. The `BlazorWebView` control can be added to any page of a .NET MAUI app, and pointed to the root of the Blazor app. The Blazor components run natively in the .NET process and render web UI to an embedded web view control. .NET MAUI Blazor apps can run on all the platforms supported by .NET MAUI.
+
+> [!IMPORTANT]
+> Using the `BlazorWebView` control requires you to have installed the ASP.NET and web development workload in Visual Studio. In addition, to use the `BlazorWebView` control on Windows requires you to install the [WebView2](https://developer.microsoft.com/microsoft-edge/webview2/) runtime.
```

Comment:
Apparently VS2022 now has WebView2 already installed as a component, so there is no extra step for this anymore.